server/config.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2018 Palantir Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"github.com/palantir/go-baseapp/baseapp"
+	"github.com/palantir/go-baseapp/baseapp/datadog"
+	"github.com/palantir/go-githubapp/githubapp"
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	DefaultAppName             = "bulldozer"
+	DefaultConfigurationV1Path = ".bulldozer.v1.yml"
```

Comment:
We decided to put the version in file name by default? Or is this left over from before `version` was a field in the configuration?

Reply:
we're starting from an opinionated (but configurable) default that the config version will be part of the path